docs/topics/Teams.md

```diff
@@ -57,11 +57,12 @@ Go team!
 
 ###### Team Object
 
-| field   | type                                                                              | description                            |
-| ------- | --------------------------------------------------------------------------------- | -------------------------------------- |
-| icon    | string                                                                            | a hash of the image of the team's icon |
-| id      | snowflake                                                                         | the unique id of the team              |
-| members | array of [team member](#DOCS_TOPICS_TEAMS/data-models-team-member-object) objects | the members of the team                |
+| field         | type                                                                              | description                            |
+| ------------- | --------------------------------------------------------------------------------- | -------------------------------------- |
+| icon          | string                                                                            | a hash of the image of the team's icon |
+| id            | snowflake                                                                         | the unique id of the team              |
+| members       | array of [team member](#DOCS_TOPICS_TEAMS/data-models-team-member-object) objects | the members of the team                |
+| owner_user_id | snowflake                                                                         | the user id of the team creator        |
```

Comment:
It's actually the id of whoever is the current owner, not just whoever created it, since ownership of teams can change. Mind changing for semantics 😄

Reply:
Good catch! Fixed in 42edf5a249cb70d6b24fc63b43d4104db33a1272